### PR TITLE
Feature-top: first section of mobile version of personal portfolio

### PR DESCRIPTION
**Pull request by kemigabocatherine**
### Mobile version homepage
**In this feature;**
- Added an HTML file for the basic structure of the page.

- Added styles using the CSS file for better appearance and design.

- Added a readme file which describes the project

### CODE REVIEW

**Review by leolpaz — CHANGES_REQUESTED** [review]
Hi @kemigabocatherine , 

You've done an amazing job so far, I'd like to highlight the following:
- Correct GitHub flow
- PR title and summary are descriptive
- Correct font was used.

Some issues need to be resolved before your approval, but you're almost there!


## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: top.html [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+        <div class="connect">
+            <p>LET’S CONNECT</p>
+        </div>
+        <br>
+        <div class="contact">
+            <ul><li><a ><img class="twitter" src="images/Twitter@2x.png" alt="Twitter"></a></li></ul>
+            <ul><li><a ><img class="linkedin" src="images/Linkedinicon@2x.png" alt="Linkedin"></a></li></ul>
+            <ul><li><a ><img class="email" src="images/Mail@2x.png" alt="email"></a></li></ul>
+            <ul><li><a ><img class="github" src="images/Github@2x.png" alt="Github"></a></li></ul>
+            <ul><li><a ><img class="vector" src="images/Vector@2x.png" alt="Vector"></a></li></ul>
```
Comment (Cathella) [inline top.html]: - [ ] As mentioned by the previous reviewer, a `<ul>` tag will be the container of all the `<li>` tags. We shall only need only one `<ul>` for this case.
- [ ] I will therefore suggest that you try to refactor the html code as shown below

```
<div class="contact">
  <ul>
    <li><a ><img class="twitter" src="images/Twitter@2x.png" alt="Twitter"></a></li>
    <li><a ><img class="linkedin" src="images/Linkedinicon@2x.png" alt="Linkedin"></a></li>

    ---- and the rest of the <li> tags ----
  </ul>
</div>
```

**Review by Cathella — CHANGES_REQUESTED** [review]
## Hi @kemigabocatherine 👋🏻, 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: top.html (hunk 1 above)
Comment (kemigabocatherine) [reply top.html]: Hi Cathella,
Thank you so much for the feedback. I am requesting that you re-review the code because I made that change already.

**Review by ShoiraTa — APPROVED** [review]
## Status: Approved :heavy_check_mark:

Hi @kemigabocatherine ,  

Your project is complete! Great job for nailing it right on the first attempt 🥇. There is nothing else to say other than... it's time to merge it 🍾🚢 :shipit: . Congratulations!  🎉💯🌟 

## Good Points 👍🏻 
- Project is set up correctly ✔️ 
- Descriptive Pull Request ✔️ 
- No linter errors ✔️ 
- Files are properly set-up ✔️ 
- Clean code ✔️ 

<br>
Cheers..Happy coding!..and keep soaring higher! 💻 🍷 🚀 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**